frontend/src/utils/utils.ts

```diff
@@ -11,6 +11,51 @@ export function differential(source: any, comparer: any): any {
   return Object.fromEntries(Object.entries(source).filter(([key, value]) => comparer[key] !== value));
 }
 
+/**
+ * @function isFileCategory
+ * Compares mimeType to a list and returns a string category for that mimetype
+ * @param {string} mimeType mimeType to be categorized
+ * @returns {string} Category of file the mimeType falls under
+ */
+export function isFileCategory(mimeType: string): string {
```

Comment:
Would rename this to getFileCategory. is implies a comparison.

Reply:
Renamed to getFileCategory.